lecture/models.py

```
@@ -24,3 +24,7 @@ def save(self, *args, **kwargs):
         if not self.slug and self.title:
             self.slug = slugify(self.title)  # Auto-generate slug from title
         super().save(*args, **kwargs)
+
+class LectureVideo(models.Model):
```

Ruff (0.8.2)

Model does not define `__str__` method

(DJ008)